osg-comanage-project-usermap.py

```diff
@@ -146,6 +149,35 @@ def get_osguser_groups(filter_group_name=None):
              for pid, gids in pid_gids.items() }
 
 
+def parse_inputfile(inputfile):
+    user_groupmap = dict()
+    with open(inputfile) as file:
```

Comment:
```suggestion
    with open(inputfile, 'r', encoding='utf-8') as file:
```

Pylint always yells at me about encoding